website/client/components/userMenu/profile.vue

```diff
@@ -229,11 +229,12 @@ div
   .nav {
     font-weight: bold;
     height: 40px;
```

Comment:
if you set this to `min-height` then it should work even better :) I could reproduce the 2nd line of nav item being not clickable, and changing the height helped

Reply:
Done.